### PR TITLE
Feature/ctype limit

### PR DESCRIPTION
**Pull request by no-reply**

### CODE REVIEW

**Comment by no-reply** [comment]
Hold off on this... need to patch the problem that caused failure on 8638276

**Comment by no-reply** [comment]
That should be better. I'm open to suggestions about how to better test this.

It may require destroying the `RDF::Marmotta` constant and reloading it to do any better than what is here.

**Comment by no-reply** [comment]
Issue submitted to Marmotta: https://issues.apache.org/jira/browse/MARMOTTA-585

**Comment by anarchivist** [comment]
Other than my point about documentation I think this is OK. :squash: if you think it's right.

File: lib/rdf/marmotta.rb [hunk 1]
```diff
@@ -14,6 +14,14 @@ class Marmotta < ::RDF::Repository
       :ldpath => 'ldpath'
     }
 
+    CTYPES = RDF::Format.content_types.select do |key, values|
```
Comment (anarchivist) [inline lib/rdf/marmotta.rb]: Can we add some documentation here with the link to the issue we filed on Marmotta? If this is fixed I'd like us to know that we should clean this up later.

**Comment by no-reply** [comment]
Comments: added. 
Branch: squashed. 
Ball: in your court. :tennis:

**Comment by anarchivist** [comment]
:basketball: :+1:

**Pull request merged**